cypress/support/pageobjects/harvest-register.js

```diff
@@ -78,6 +78,50 @@ export class HarvestRegister{
     deleteHarvestReportItem(){
         return cy.get('a.scp-registered-date.scp-harvest-date.selected').prev()
     };
+    confirmDeleteReportButton(){
+        cy.wait(500)
```

Comment:
Why do you need this wait? cy.get waits for element. e.g.
// waits 10s for the element
cy.get('#some-delayed-element', { timeout: 10000 }).click();